region/lowernorfair/west.json

```diff
@@ -782,11 +806,68 @@
                   "canHeatRun"
                 ]},
                 {"or": [
+                  {"obstacle":{
+                    "id": "A",
+                    "requires": [
+                      "canIBJ",
```

Comment:
Since going 3 -> 2 breaks obstacle 'A', going back up using what used to be this IBJ obstacle req is a free null req? Am I understanding obstacles wrong?

Reply:
No, that's me being stupid. Wow, that would've been bad.